llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -201,6 +196,10 @@ cl::opt<module_split::IRSplitMode> SplitMode(
 cl::opt<bool> DoSymGen{"symbols", cl::desc("generate exported symbol files"),
                        cl::cat(PostLinkCat)};
 
+cl::opt<bool> DoPropGen{"properties",
```

Comment:
We have some checks in place for DoSymGen (towards the end of this file. Think we will need to add similar checks for this as well?

Thanks

Reply:
good catch, fixed in latest commit, thank you!